lib/browser/rpc-server.js

```diff
@@ -449,3 +450,21 @@ ipcMain.on('ELECTRON_BROWSER_WINDOW_CLOSE', function (event) {
   }
   event.returnValue = null
 })
+
+ipcMain.on('ELECTRON_BROWSER_SANDBOX_LOAD', function (event, preloadPath) {
+  let preloadSrc = null
+  if (preloadPath) {
+    try {
+      preloadSrc = fs.readFileSync(preloadPath).toString()
+    } catch (err) {
+      preloadSrc = err
```

Comment:
maybe a separate variable? preloadError?

Reply:
Also we need to be aware the `Error` objects to not JSON stringify very well by default, you'll need to manually pull out the `stack` and `message` attributes into a new object